keyboards/kumo/keymaps/LEdiodes/keymap.c

```diff
@@ -0,0 +1,211 @@
+ * `----------------------------------------------------------------------------'
+ */
+[L2] = LAYOUT(____, ____, ____, ____, ____, ____, ____, KC_PGUP, KC_UP, KC_PGDN, ____, ____, \
+              KC_CAPS, ____, ____, ____, ____, ____, KC_HOME, KC_LEFT, KC_DOWN, KC_RGHT, ____, ____, \
+              ____, ____, ____, ____, ____, ____, KC_END, ____, ____, ____, M(thumbup), ____, \
+              RESET, ____, M(cad), KC_NO, ____, ____, ____, M(snip), M(handsh), M(note) \
+             ),
+
+/* Green Layer 3 - Media Controls
+ * ,---------------------------------------------------------------------------.
+ * |     |     |     |     |     |     |     |     |     |     |     |   DEL   |
+ * |---------------------------------------------------------------------------+
+ * |      |     |     |     |     |     |     |     |     |     |     |  VOL+  |
+ * |---------------------------------------------------------------------------+
+ * |         |     |     |     |     |     |     |     |     |MUTE |STOP |VOL- |
+ * |---------------------------------------------------------------------------+
+ * |RESET |        |      |             |            |   L2  |PREV |PLAY |NEXT |
+ * `---------------------------------------------------------------------------'
+ */
+[L3] = LAYOUT(____, ____, ____, ____, ____, ____, ____, ____, ____, ____, ____, KC_DEL, \
+              ____, ____, ____, ____, ____, ____, ____, ____, ____, ____, ____, KC_VOLU, \
+              ____, ____, ____, ____,____, ____, ____, ____, ____, KC_MUTE, KC_MSTP, KC_VOLD, \
+              RESET,  ____, ____, KC_NO, ____, ____, ____, KC_MPRV, KC_MPLY, KC_MNXT \
+             ),
+
+/* Green Layer 4 - Mouse and Arrow Control
+ * ,---------------------------------------------------------------------------.
+ * | BTN1|     |     |     |     |     |     |BTN1 | M-U |BTN2 |     |  BTN2   |
+ * |---------------------------------------------------------------------------+
+ * |      |     |     |     |     |     |     | M-L | M-D | M-R |     |        |
+ * |---------------------------------------------------------------------------+
+ * |         |     |     |     |     |     |     |     |     |     | UP  |     |
+ * |---------------------------------------------------------------------------+
+ * |RESET |        |      |             |            |       |LEFT |DOWN |RIGHT|
+ * `---------------------------------------------------------------------------'
+ */
+[L4] = LAYOUT(KC_BTN1, ____, ____, ____, ____, ____, ____, KC_BTN1, KC_MS_U, KC_BTN2, ____, KC_BTN2, \
+              ____, ____, ____, ____, ____, ____, ____, KC_MS_L, KC_MS_D, KC_MS_R, ____, ____, \
+              ____, ____, ____, ____,____, ____, ____, ____, ____, ____, KC_UP, ____, \
+              RESET,  ____, ____, KC_NO, ____, ____, ____, KC_LEFT, KC_DOWN, KC_RIGHT \
+             ),
+};
+// Macros
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`.